features/step_definitions/pull_request_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^I see a new (.+?) pull request for the "(.+?)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
+  expect(@last_run_result.out).to eql "#{@tool} called with: #{pull_request_url domain, branch, repo}\n"
+end
```

Comment:
To me, these all deal with the output of running a git-town command. The object to me in each case is the output (even though its not explicitly stated).